Client/Frontend/UserContent/UserScripts/AllFrames/AtDocumentEnd/__firefox__.js

```diff
@@ -10,19 +10,82 @@ if (!window.__firefox__) {
     enumerable: false,
     configurable: false,
     writable: false,
-    value: {
-      userScripts: {},
-      includeOnce: function(userScript, initializer) {
-        if (!__firefox__.userScripts[userScript]) {
-          __firefox__.userScripts[userScript] = true;
-          if (typeof initializer === 'function') {
-            initializer();
-          }
-          return false;
-        }
+    value: (function() {
```

Comment:
nit: I'm not sure I understand why we need `__firefox__.js` added at both `AtDocumentStart` and `AtDocumentEnd` but we can file a follow-up issue to figure this out.